script.js

```diff
@@ -0,0 +1,63 @@
+// Collection of books
+let books = [];
+
+// Load books from local storage
+const loadBooks = () => {
+  const storedBooks = localStorage.getItem('books');
+  if (storedBooks) {
+    books = JSON.parse(storedBooks);
+  }
+};
+
+// Save books to local storage
+const saveBooks = () => {
+  localStorage.setItem('books', JSON.stringify(books));
+};
+
+// Display books in the collection
+const displayBooks = () => {
+  const bookCollection = document.querySelector('#book-collection');
+  bookCollection.innerHTML = '';
+
+  books.forEach((book) => {
+    const bookEl = document.createElement('div');
+    bookEl.classList.add('book');
+    bookEl.innerHTML = `
+      <span>${book.title} <br> ${book.author}</span>
+      <button data-index="${book.index}">Remove</button>
+    `;
+    bookCollection.appendChild(bookEl);
+  });
+};
+
+// Add a book to the collection
+const addBook = () => {
+  const title = document.querySelector('#title').value;
+  const author = document.querySelector('#author').value;
+  books.push({
+    index: books.length,
+    title,
+    author,
+  });
+  saveBooks();
+  displayBooks();
+};
```

Comment:
- [ ] [OPTIONAL] It would be better if you empty the input fields once the add button is clicked, otherwise the user will have to remove the previously added book in order to add a new one, this will improve the UI experience

**hint:** 
```
const emptyForm = () => {
  document.querySelector('#title').value = ''
  document.querySelector('#author').value = ''
}
```